app/src/main/res/values/strings.xml

```diff
@@ -204,68 +203,68 @@
     <string name="dialog_delete_selected">Delete selected</string>
     <string name="explore_folder_add_sudoku">Add sudoku</string>
     <string name="edit_sudoku">Edit sudoku</string>
     <plurals name="delete_selected_in_folder">
-        <item quantity="one">Are you sure you want to delete %1$d game?</item>
-        <item quantity="two">Are you sure you want to delete %1$d games?</item>
-        <item quantity="few">Are you sure you want to delete %1$d games?</item>
-        <item quantity="many">Are you sure you want to delete %1$d games</item>
-        <item quantity="other">Are you sure you want to delete %1$d games?</item>
+        <item quantity="one">Delete %1$d game?</item>
+        <item quantity="two">Delete %1$d games?</item>
+        <item quantity="few">Delete %1$d games?</item>
+        <item quantity="many">Delete %1$d games</item>
+        <item quantity="other">Delete %1$d games?</item>
     </plurals>
     <string name="game_id">Game #%1$d</string>
     <string name="add_to_folder">Add to folder</string>
     <string name="add_to_folder_create_new">Create new</string>
     <string name="add_to_folder_from_file">Import from a file</string>
-    <string name="last_played_section_title">Recently played games</string>
+    <string name="last_played_section_title">Recently played</string>
     <string name="pref_boardtheme_accent">Apply accent colors</string>
-    <string name="pref_boardtheme_accent_subtitle">Make your sudoku board more colorful</string>
-    <string name="pref_board_theme_title">Sudoku board theme</string>
-    <string name="pref_board_theme_subtitle">Customize the look of the sudoku board</string>
+    <string name="pref_boardtheme_accent_subtitle">Colorize the board</string>
+    <string name="pref_board_theme_title">Board theme</string>
+    <string name="pref_board_theme_subtitle">Customize the look of the board</string>
     <string name="board_theme_title">Board theme</string>
     <string name="action_move_selected">Move selected</string>
-    <string name="move_games_to_folder_subtitle">Select the folder to which you want to move the selected games</string>
+    <string name="move_games_to_folder_subtitle">Select a folder to move the selected games to</string>
     <string name="pref_cross_highlight_subtitle">Highlight some boxes on the board</string>
     <string name="pref_cross_highlight">Cross highlight</string>
-    <string name="dialog_crash_report_text">Oops, LibreSudoku just crashed. Do you want to send a crash report to the developers?</string>
+    <string name="dialog_crash_report_text">LibreSudoku crashed. Send a report to the developers?</string>
     <string name="pref_crash_reporting">Crash reporting</string>
-    <string name="pref_crash_reporting_subtitle">Help us fix bugs. App doesn\'t send logs by itself, you will be prompted to send a crash report by email</string>
+    <string name="pref_crash_reporting_subtitle">Ask to submit bug reports via e-mail.</string>
```

Comment:
```suggestion
    <string name="pref_crash_reporting_subtitle">Submit bug reports via e-mail.</string>
```
The prior text made a point out of it being a prompt. I am all about making it shorter, but thought it could be misleading to omit that.